javascript/big-practice/src/scripts/models/noteModel.js

```diff
@@ -106,28 +75,18 @@ export default class NoteModel {
    * @return {Object} this.notes[noteIndex]
    */
   async deleteNote(id) {
-    let noteItem = {};
-    const date = new Date().toISOString().slice(0, 10);
-
-    /**
-     * This condition check if userId is available in localStorage.
-     * It will change field of note in JSON
-     * if userId is not available, it will change field in localStorage
-     */
-    if (!this.localStorage.getItems(STORAGE_KEYS.USER_ID)) {
-      const noteIndex = this.notes.findIndex((note) => note.id === id);
-      this.notes[noteIndex].deleteAt = date;
-
-      this.localStorage.setItems(STORAGE_KEYS.LIST_NOTE, this.notes);
-      noteItem = this.notes[noteIndex];
-    } else {
-      noteItem = await getDataById(id);
+    try {
+      const date = new Date().toISOString().slice(0, 10);
+      const noteItem = this.listNotes.find((note) => note.id === id);
+
       noteItem.deleteAt = date;
+      await putNote(id, noteItem);
 
-      putData(id, noteItem);
+      return noteItem;
+    } catch (error) {
+      console.log(error);
+      throw error;
     }
-
-    return noteItem;
   }
 
   /**
```

Comment:
It would be best if you made a clearer description of each method in the model. This method will remove the item from the database, right?